usagov/vote-gov-drupal
# VOTE-1115 update admin listings

Exposed all translation options to listing pages that apply Update admin listing page titles to be explicit

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1115

## Description (optional)
Update admin listings for translations

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. delete block content: Accordion Group, Basic & Registration tool
2. Uninstall inline_entity_form
3. Uninstall migrate module and related migration modules

### Post-deploy steps
1. run `lando retune`

### Testing steps
1. Visit http://vote-gov.lndo.site/admin/content and make sure the translation filter has all options visible.
2. Visit http://vote-gov.lndo.site/admin/content/media and make sure the translation filter has all options visible.
